<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-GUK9D9E" failures="10" tests="10" name="TestF" time="69.962" errors="0" timestamp="2021-01-20T12:53:16 IST">
  <testcase classname="Tests.FunctionalityTests" name="autoAddingQuestionMark" time="0.007">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Tests.FunctionalityTests.autoAddingQuestionMark(FunctionalityTests.java:87)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- autoAddingQuestionMark -->
  <testcase classname="Tests.FunctionalityTests" name="backToThePreviousPageOne" time="11.566">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Tests.FunctionalityTests.backToThePreviousPageOne(FunctionalityTests.java:167)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- backToThePreviousPageOne -->
  <testcase classname="Tests.FunctionalityTests" name="backToThePreviousPageTwo" time="0.003">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Tests.FunctionalityTests.backToThePreviousPageTwo(FunctionalityTests.java:179)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- backToThePreviousPageTwo -->
  <testcase classname="Tests.FunctionalityTests" name="bounderValuesMax" time="10.378">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Tests.FunctionalityTests.bounderValuesMax(FunctionalityTests.java:75)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- bounderValuesMax -->
  <testcase classname="Tests.FunctionalityTests" name="bounderValuesMin" time="11.67">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Tests.FunctionalityTests.bounderValuesMin(FunctionalityTests.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- bounderValuesMin -->
  <testcase classname="Tests.FunctionalityTests" name="errorHandling31CharsInAnswer" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Tests.FunctionalityTests.errorHandling31CharsInAnswer(FunctionalityTests.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- errorHandling31CharsInAnswer -->
  <testcase classname="Tests.FunctionalityTests" name="errorHandling51Chars" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Tests.FunctionalityTests.errorHandling51Chars(FunctionalityTests.java:136)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- errorHandling51Chars -->
  <testcase classname="Tests.FunctionalityTests" name="errorHandling51CharsInQuestion" time="9.535">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Tests.FunctionalityTests.errorHandling51CharsInQuestion(FunctionalityTests.java:118)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- errorHandling51CharsInQuestion -->
  <testcase classname="Tests.FunctionalityTests" name="errorHandlingHeb" time="9.858">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Tests.FunctionalityTests.errorHandlingHeb(FunctionalityTests.java:104)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- errorHandlingHeb -->
  <testcase classname="Tests.FunctionalityTests" name="quitTheTriviaGame" time="11.213">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Tests.FunctionalityTests.quitTheTriviaGame(FunctionalityTests.java:151)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- quitTheTriviaGame -->
</testsuite> <!-- TestF -->
